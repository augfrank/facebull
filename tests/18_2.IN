M0   C0   C2  2496
M1   C0   C3  2749
M2   C0   C9  2475
M3   C0   C10  2103
M4   C0   C15  2697
M5   C1   C0  2345
M6   C1   C3  2396
M7   C1   C5  2516
M8   C1   C7  2870
M9   C1   C9  2523
M10   C1   C10  2082
M11   C2   C6  2491
M12   C2   C8  2474
M13   C2   C16  2976
M14   C2   C17  2844
M15   C3   C2  2353
M16   C3   C4  2477
M17   C3   C7  2880
M18   C3   C8  2617
M19   C4   C6  2836
M20   C4   C10  2535
M21   C4   C13  2531
M22   C4   C16  2841
M23   C5   C0  2504
M24   C5   C6  2490
M25   C6   C4  2838
M26   C6   C9  2468
M27   C6   C12  2257
M28   C6   C17  2418
M29   C7   C0  2983
M30   C7   C1  2400
M31   C7   C4  2064
M32   C7   C8  2493
M33   C7   C9  2266
M34   C7   C10  2227
M35   C7   C16  2062
M36   C8   C2  2836
M37   C8   C3  2221
M38   C8   C4  2409
M39   C8   C5  2566
M40   C8   C10  2361
M41   C8   C11  2786
M42   C8   C14  2256
M43   C8   C16  2477
M44   C8   C17  2571
M45   C9   C1  2587
M46   C9   C6  2173
M47   C9   C7  2822
M48   C9   C12  2014
M49   C9   C14  2873
M50   C9   C15  2851
M51   C10   C11  2874
M52   C10   C13  2217
M53   C10   C14  2773
M54   C11   C0  2460
M55   C11   C1  2465
M56   C11   C7  2613
M57   C11   C14  2081
M58   C11   C16  2165
M59   C12   C2  2367
M60   C12   C4  2962
M61   C12   C7  2651
M62   C12   C11  2165
M63   C12   C14  2867
M64   C13   C5  2384
M65   C13   C6  2729
M66   C13   C7  2897
M67   C13   C11  2865
M68   C14   C7  2346
M69   C14   C10  2747
M70   C15   C5  2847
M71   C15   C7  2460
M72   C15   C13  2098
M73   C16   C2  2608
M74   C16   C3  2598
M75   C16   C5  2190
M76   C16   C9  2313
M77   C16   C15  2309
M78   C17   C0  2507
M79   C17   C4  2693
M80   C17   C7  2205
M81   C17   C15  2050
