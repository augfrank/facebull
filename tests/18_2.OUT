42163
3 6 12 15 19 23 28 35 38 48 53 55 62 64 68 72 76 81
